BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -108,7 +108,7 @@
   <ul class="tsk-prm-list">
     <li>
       <%%
-        input.(id, name,
+        input.(id, name+"[]",
```

Comment:
Be careful. In rails parameter names determine how their VALUES will be pre-processed...

```html
<input name="abc" value="val">
```

gives

```ruby
params =  { :abc => "val" }
```

while

```html
<input name="abc[]" value="val">
```

gives

```ruby
params = { :abc => [ "val" ] }
```